content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -93,192 +132,209 @@ product that this newly-created serial number was just assigned to.
-   :doc:`create_sn`
+- **Incoming goods:** Assign serial numbers directly on the **receipt**. Receipts can be accessed by
+  navigating to :menuselection:`Inventory app --> Operations --> Receipts`.
+- **Outgoing goods:** Assign serial numbers directly on the **delivery order**. Receipts can be
+  accessed by navigating to :menuselection:`Inventory app --> Operations --> Deliveries`.
 
-.. image:: serial_numbers/assign-serial-numbers.png
+.. important::
+   Before assigning serial numbers on receipts or delivery orders, be sure that the ability to
+   :ref:`create new serial numbers by operations type
+   <inventory/product_management/operation-type-setting>` is enabled.
+
+Lots/serial number field
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Serial numbers can be entered directly into the Serial Numbers field on a receipt or delivery order.
+
+.. image:: serial_numbers/enter-in-field.png
    :align: center
-   :alt: Display the bulleted list icon and Detailed Operations smart buttons.
+   :alt: Select value for Serial Number field on receipt.
 
-.. warning::
-   Clicking :guilabel:`Validate` before assigning a serial number to received quantities will cause
-   a :guilabel:`User Error` pop-up to appear. The pop-up requires entry of a lot or serial number
-   for the ordered products. The :abbr:`RFQ (request for quotation)` **cannot** be validated without
-   a serial number being assigned.
+.. important::
+   To make the Serial Numbers field visible on a receipt or delivery order, click the
+   :icon:`oi-settings-adjust` :guilabel:`(Adjust Settings)` icon, and ensure the box next to Serial
+   Numbers is checked.
 
-   .. image:: serial_numbers/serial-numbers-user-error-popup.png
+   .. figure:: serial_numbers/field-visible.png
       :align: center
-      :alt: User error popup prompting serial number entry.
+      :alt: Allow Serial Numbers field to show on a receipt or delivery order.
 
-There are multiple ways to do this: manually assigning serial numbers, automatically assigning
-serial numbers, and copy/pasting serial numbers from a spreadsheet.
+.. _inventory/product_management/stock-move:
 
-Assign serial numbers automatically
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Stock move pop-up window
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-If a large quantity of products need individual serial numbers assigned to them, Odoo can
-automatically generate and assign serial numbers to each of the individual products.
+For various methods of assigning serial numbers individually or in bulk, click the
+:guilabel:`⦙≣ (bulleted list)` icon in the product line of a receipt.
 
-To accomplish this, start with the :guilabel:`First SN` field in the :guilabel:`Detailed Operations`
-pop-up window, and type the first serial number in the desired order to be assigned.
+Add a line
+**********
+
+In the :guilabel:`Open: Stock move` pop-up window that appears, serial numbers can be manually input
+in the :guilabel:`Lot/Serial Number` column. This method is best reserved for adding only one or a
```

Comment:
Optional suggestion for brevity
```suggestion
In the :guilabel:`Open: Stock move` pop-up window, manually enter serial numbers 
in the :guilabel:`Lot/Serial Number` column. This method is best reserved for adding only one or a
```

Reply:
Or, we can elaborate/remind the reader how they open this popup. 

```suggestion
In the :guilabel:`Open: Stock move` pop-up window that appears after clicking the :icon:`fa-list` :guilabel:`(list)` icon in the product line, serial numbers can be manually input
in the :guilabel:`Lot/Serial Number` column. This method is best reserved for adding only one or a
```

Reply:
I chose the first option because the pop-up opening instructions are like two lines before this one!